.ci/teardown.sh

```diff
@@ -17,22 +17,53 @@
 # This script will get any execution log that may be useful for
 # debugging any issue related to Clear Containers.
 
-runtime_log_file="/var/lib/clear-containers/runtime/runtime.log"
+log_copy_dest="$1"
 
-echo "Clear Containers Runtime Log:"
-sudo cat "$runtime_log_file"
+runtime_log_location="/var/lib/clear-containers/runtime/runtime.log"
+runtime_log_filename="cc-runtime.log"
+runtime_log_path="${log_copy_dest}/${runtime_log_filename}"
 
-if [[ ! $(ps -p 1 | grep systemd) ]]; then
-	upstart_logs_path="/var/log/upstart"
-	echo "Clear Containers Proxy Log:"
-	sudo cat "${upstart_logs_path}/cc-proxy.log"
-	echo "CRI-O Log:"
-	sudo cat "${upstart_logs_path}/crio.log"
+proxy_log_filename="cc-proxy.log"
+proxy_log_path="${log_copy_dest}/${proxy_log_filename}"
+
+shim_log_filename="cc-shim.log"
+shim_log_path="${log_copy_dest}/${shim_log_filename}"
+
+crio_log_filename="crio.log"
+crio_log_path="${log_copy_dest}/${crio_log_filename}"
+
+upstart_logs_path="/var/log/upstart"
+upstart_proxy_path="${upstart_logs_path}/${proxy_log_filename}"
+upstart_crio_path="${upstart_logs_path}/${crio_log_filename}"
+
+# Copy log files if a destination path is provided, otherwise simply
+# display them.
+if [ ${log_copy_dest} ]; then
+	sudo cp "${runtime_log_location}" "${runtime_log_path}"
+
+	if [[ -n $(ps -p 1 | grep systemd) ]]; then
+		sudo journalctl --no-pager -u cc-proxy > "${proxy_log_path}"
+		sudo journalctl --no-pager -t cc-shim > "${shim_log_path}"
+		sudo journalctl --no-pager -u crio > "${crio_log_path}"
+	else
+		sudo cp "${upstart_proxy_path}" > "${proxy_log_path}"
+		sudo cp "${upstart_proxy_path}" > "${crio_log_path}"
```

Comment:
Is this meant to be crio_path, not proxy_path? (copy/paste error?). Or at least not copying the proxy path to the crio log path :-)

Reply:
oh yeah I did a mistake here !